roles/replace_node/README.md

```diff
@@ -0,0 +1,61 @@
+Role Name
+=========
+
+replace_node - Replace gluster node with a new node.
+
+Requirements
+------------
+
+- Ansible >= 2.6
+- lvm2 (Optional)
+
+
+NOTE: Inventory should contain a single node, which should be part of the cluster. This node should be different than any of the three variables explained below, this requirement is to collect the right peers for the new node.
+
+Role Variables
+--------------
+
```

Comment:
Repeated content? Same comments apply as in README.md. Should that README link to this one instead?

Reply:
Actually the plan is, when we add more sub-roles like `replace-brick', `replace-ip' etc... the README.md at the top level will have superset of these variables. For example: https://github.com/gluster/gluster-ansible-features/blob/master/README.md